gcloud-java-storage/src/main/java/com/google/gcloud/spi/DefaultStorageRpc.java

```diff
@@ -190,6 +192,12 @@ public Bucket get(Bucket bucket, Map<Option, ?> options) {
           .setIfMetagenerationNotMatch(IF_METAGENERATION_NOT_MATCH.getLong(options))
           .setFields(FIELDS.getString(options))
           .execute();
+    } catch (GoogleJsonResponseException ex) {
+      GoogleJsonError jsonError = ex.getDetails();
```

Comment:
Though slightly less "optimize" I prefer the style that we did for `delete`.

```java
  } catch (IOException ex) {
      StorageException serviceException = translate(ex);
      if (serviceException.code() == 404) {
        return false;
      }
      throw serviceException;
    }
```

Regardless of my preference, can you make them consistent and also replace the `delete` 404 with HTTP_NOT_FOUND?